src/utils/__tests__/prepareBlocks.test.js

```diff
@@ -64,4 +64,61 @@ describe('createBlocks', () => {
       }).preparedBlocks,
     ).toHaveLength(3);
   });
+
+  it('should create empty release for teams with channels when there are no messages', () => {
+    const hasMessages = false;
```

Comment:
Sorry! 🙇 I think this may be more clear:

```suggestion
    const hasNoMessages = false;
```